### ISSUE TITLE
All dropdowns close automatically

### ISSUE THREAD

**Issue by Jameskmonger** [sindresorhus/refined-github#28]
I've noticed that the dashboard context switcher closes automatically for about 5 seconds after the page has loaded. After that time, it opens and closes as expected.

**Comment by Jameskmonger** [comment]
This appears to be a duplicate of #19 by @hkdobrev but with a different resolution method (click and hold vs wait).

@sindresorhus, would you like me to close this?

**Comment by Jameskmonger** [comment]
This doesn't appear to just affect the dashboard context as described in #19. It also affects the profile drop down in the top right.

**Comment by paulmolluzzo** [comment]
@jameskmogner I think you're right and this can be closed as duplicate. PR #27 might be a fix but waiting on some feedback/testing for a merge. Maybe give that a shot?

**Comment by paulmolluzzo** [comment]
The underlying cause is the `btn.click()` firing every 250ms until you're entire feed history is loaded. For an old account it'll last a while. It doesn't matter what dropdown you use, it's clicking outside of the dropdown in that "More" button, which is why they get closed. It's a duplicate _cause_ even if it's a different effect than originally described.
